docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -37,7 +37,7 @@ a simple way of running Kubernetes on your local workstation for free.
 ## Create a Minikube cluster
 
 This tutorial uses [Minikube](https://github.com/kubernetes/minikube) to 
-create a local cluster. It assumes you are using [Docker for Mac](https://docs.docker.com/engine/installation/mac/)
+create a local cluster. This tutorial also assumes you are using [Docker for Mac](https://docs.docker.com/engine/installation/mac/)
```

Comment:
Thanks, this makes a little more sense. If you're going to tell people to use the --vm-driver flag though, they still need to install the xhyve driver from my other comment. Could you add that here?